### PR TITLE
dbc files updated BattlemasterList.dbc, Stationery.dbc, ServerMessages.dbc

### PR DESCRIPTION
**Pull request by schnek**

### CODE REVIEW

File: src/world/Management/MailMgr.h [hunk 1]
```diff
@@ -185,4 +188,4 @@ class SERVER_DECL MailSystem : public EventableObject
 
 #define MAIL_DEFAULT_EXPIRATION_TIME 30
 
-#endif // MAILMGR_H
+#endif // MAILMGR_H
```
Comment (Appled) [inline src/world/Management/MailMgr.h]: Why removing newline?

Comment (schnek) [reply src/world/Management/MailMgr.h]: up

**Comment by Zyres** [comment]
change the target branch to develop

**Comment by schnek** [comment]
up

**Pull request merged**